src/components/Gauge.js

```diff
@@ -17,6 +17,7 @@ const Gauge = React.createClass({
     dataPointRadius: React.PropTypes.number,
     dataPoints: React.PropTypes.array,
     defaultLabelText: React.PropTypes.string,
+    defaultLabelValueColor: React.PropTypes.string,
```

Comment:
I would also change the name of this prop to `defaultLabelColor` as well.  Adding the word `value` to it makes it read weird.